google/cloud/networkconnectivity/README.md

```diff
@@ -1,10 +1,12 @@
 # Network Connectivity API C++ Client Library
 
 This directory contains an idiomatic C++ client library for the
-[Network Connectivity API][cloud-service-docs], a service to The Network Connectivity API provides access to Network Connectivity Center.
+[Network Connectivity API][cloud-service-docs], a suite of products that provide
+enterprise connectivity from your on-premises network or from another cloud
+provider to your Virtual Private Cloud (VPC) network.
 
-While this library is **GA**, please note that the Google Cloud C++ client libraries do **not** follow
-[Semantic Versioning](https://semver.org/).
+While this library is **GA**, please note that the Google Cloud C++ client
+libraries do **not** follow [Semantic Versioning](https://semver.org/).
```

Comment:
I changed my generator to add the line break here. I didn't feel like regenerating the files.